ParserHelpers.cpp

```diff
@@ -92,6 +92,43 @@ void ignoreString(const std::string& unused, std::istream& theStream)
 }
 
 
+
+template <typename T>
+std::enable_if_t<std::is_integral_v<T>, T> stringToInteger(const std::string& str, bool skipPartialMatchWarning) // for integral types only
+{
+	T theInteger{0};
```

Comment:
Is ```T theInteger = 0;``` possible here? I find that {0} looks like a container is being given the single element 0.